src/Lecture_3/Homework_3/Task_4.java

```diff
@@ -0,0 +1,35 @@
+package Lecture_3.Homework_3;
+
+import java.util.Arrays;
+
+public class Task_4 {
+
+    public static void main(String[] args) {
+
+        int [] array = {23, 4, 4, 23};
+        int [] reversedArray = new int [array.length];
+
+        int length = array.length;
+        int start = 0;
+        int end = length - 1;
+
+        for (int i = 0; i < array.length; i++) {
```

Comment:
You don't end extra array.
You could do the check by using only one loop.

Reply:
Got it, changed code to not to use second array.